string name
byte initialized
float64 position
float64 velocity
float64 applied_effort
float64 commanded_effort